### PR TITLE
mtr: update page

### PR DESCRIPTION
**Pull request by marek22k**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 0.94

### CODE REVIEW

File: pages/common/mtr.md [hunk 1]
```diff
@@ -5,24 +5,28 @@
 
 - Traceroute to a host and continuously ping all intermediary hops:
 
-`mtr {{host}}`
+`mtr {{example.com}}`
```
Comment (mfrw) [inline pages/common/mtr.md:8]: As per the title (which is long and maybe we could rephrase it to something more concise), we might have missed to update the homepage.

Comment (marek22k) [reply pages/common/mtr.md:8]: Oops, I guess I forgot to push the change. Feel free to suggest something "better" for the title.

File: pages/common/mtr.md [hunk 2]
```diff
@@ -1,28 +1,32 @@
 # mtr
 
 > Matt's Traceroute: combined traceroute and ping tool.
-> More information: <https://bitwizard.nl/mtr>.
+> More information: <https://www.bitwizard.nl/mtr/>.
```
Comment (kbdharun) [inline pages/common/mtr.md:4]:
```suggestion
> More information: <https://bitwizard.nl/mtr/>.
```
The URL seems to work without `www` too so maybe we can skip its addition.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/mtr.md (hunk 2 above)
Comment (marek22k) [reply pages/common/mtr.md:4]: On https://github.com/traviscross/mtr it is linked to www and since that is the official git repo, I took it that way.

**Review by mfrw — COMMENTED** [review]
LGTM
Thank you for your contribution @marek22k

**Review by mfrw — APPROVED** [review]
